lib/i18n/_missing_translations.json

```diff
@@ -20,13 +20,6 @@
   "es": {
     "update": {
       "updateAvailableDescription(OUTDATED)": "A new version of the app is available, with these changes:"
```

Comment:
Please can you also translate this one (in a new PR)?

Reply:
Ok